# Generated by Django 4.1.5 on 2023-02-05 02:15

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ("beacon", "0010_rename_exception_log_message_and_more"),
    ]

    operations = [
        migrations.CreateModel(
            name="ImportLog",
            fields=[
                (
                    "id",
                    models.BigAutoField(
                        auto_created=True,
                        primary_key=True,
                        serialize=False,
                        verbose_name="ID",
                    ),
                ),
                ("log_type", models.CharField(max_length=20)),
                ("name", models.CharField(max_length=50)),
                ("message", models.TextField()),
                ("status", models.IntegerField(default=0)),
                ("date_time", models.DateTimeField(auto_now_add=True, null=True)),
            ],
        ),
        migrations.AddField(
            model_name="log",
            name="date_time",
            field=models.DateTimeField(auto_now_add=True, null=True),
        ),
        migrations.AlterField(
            model_name="scenario",
            name="modify_date",
            field=models.DateTimeField(
                blank=True, default="2023-02-05 02:15:29", null=True
            ),
        ),
    ]
